### ISSUE TITLE
Improve grey text readability (may be hard to read for vision impaired readers)

### ISSUE THREAD

**Issue by sjoerdly** [ScrumFacilitators/measuringoutcome-cht#3]
First off, I love the idea of putting explanations on the cards. And having both chinese and english on there so they can also serve as a translation map when reading other EBM related materials (bonus feature!)

The grey text may be hard to read on some of the backgrounds, especially for visually impaired readers.

Suggestion: make grey text a few shades whiter, so there is a distinction between the white and (lighter) grey, but the readability improves. 

This blog has a list of tools that may be of use to get this right https://www.accessiblemetrics.com/blog/7-web-accessibility-testing-tools-for-every-need/

**Comment by ctonylee** [comment]
@sjoerdly, 

1) thank you for this great suggestion about readability.
I've noticed the same problem readability issue after the first release. 
I am working on an update including updating the color from grey to white. 

> putting explanations on the cards.
2) And thank you for the positive comment about changing the design. 
From a discussion with @cheehonghsia , I realized that I forgot to check with SF before making the change. 
And the CHT cards now have a different design from other languages...
Hence, I was just about to revert this design to keep its consistency.
What are your thoughts on this? 
And any suggestions for how future design change workflow should be conducted? 


![image](https://user-images.githubusercontent.com/1815759/103255684-35818e00-49c5-11eb-8d45-5f7ee6771d41.png)

**Comment by sjoerdly** [comment]
Hi Tony, awesome that you picked this up so diligently. 

I've discussed (briefly) with @cheehonghsia about question 2. 

>And the CHT cards now have a different design from other languages...
> Hence, I was just about to revert this design to keep its consistency.
> What are your thoughts on this?
> And any suggestions for how future design change workflow should be conducted?

The CC-BY-SA-4.0 license is pretty relaxed, and from that standpoint you could change anything as long as you refer back to us and the original english version. 

As this translation is hosted on our SF repo, it would be good that the design style (coloring, main layout of edges etc) is still recognizable as SF. Other than that, you can 'do first, discuss later' based on your own awesomeness and insight. 

In your example, the design changes stay well within the boundaries of 'recognizable as SF'. So rock on!

**Comment by ctonylee** [comment]
closing issue because PR https://github.com/ScrumFacilitators/measuringoutcome-cht/pull/5 has updated the text color
